packages/coin-base/src/derive/ed25519.ts

```
@@ -0,0 +1,111 @@
+    const randBytes = base.randomBytes(32);
+    if (signUtil.ed25519.privateKeyVerify(randBytes)) {
+      if (ed25519SignTest(randBytes)) {
+        const publicKey = signUtil.ed25519.publicKeyCreate(randBytes);
+        const privateKey: Uint8Array = concatPub ? base.concatBytes(randBytes, publicKey) : randBytes;
+        return encode === 'base58' ? base.toBase58(privateKey) : base.toHex(privateKey);
+      }
+    }
+  }
+}
+
+const pathRegex = new RegExp("^m(\\/[0-9]+')+$");
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)